debian/control

```diff
@@ -0,0 +1,15 @@
+Source: python3-templated-dictionary
+Section: python
+Priority: optional
+Maintainer: Miroslav Suchý <msuchy@redhat.com>
```

Comment:
I really do not want to be listed here as a Debian maintainer. See https://www.debian.org/doc/debian-policy/ch-binary.html#s-maintainer
I do not have enough of the Debian packaging wisdom and I cannot answer questions about the Debian packaging of this library.

Reply:
it's just put for it's change in the final package

see yourself on lauchpad

https://launchpad.net/~andykimpe/+archive/ubuntu/mock/+packages

Changelog

distribution-gpg-keys (1.72-2~focal) focal; urgency=medium

  * just update changelog for gpg sign and add ppa.

 -- Kimpe Andy <andykimpe@gmail.com> Wed, 29 Jun 2022 18:59:14 +0000

code examples must always contain the contact of the original developer of the project and not that of the packager

the packager changes it himself when making the package

the rules are that if you provided sample code in the original project.

the example should always be with the information of the original developer of the project and not that of the packager.

the goal here is that once the ppa and dpa deposits are 100% complete.

Have sent a request to debian and canonical for who adds the packages in

ubuntu multiverse

and

debian contrib

on the official deposits if the rules which impose are not scrupulously respected the request will be automatically rejected

if it imposes it is rule it is by what is need of the contact of the original developer of the project in case there would be a problem with the project itself and not with the packaging